documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -14,290 +14,210 @@ keywords:
-
-Example of a Diff DID message:
-```json
-{
-  "id": "did:iota:X7U84ez4YeaLwpfdnhdgFyPLa53twvAuMSYdRQas54e",
-  "diff": {
-    "doc": {
-      "service": [
-        {
-          "id": "did:iota:X7U84ez4YeaLwpfdnhdgFyPLa53twvAuMSYdRQas54e#linked-domain-1",
-          "type_": "LinkedDomains",
-          "service_endpoint": "https://example.com/"
-        }
-      ]
-    },
-    "meta": {
-      "updated": "2022-01-21T09:50:28Z"
-    }
-  },
-  "previousMessageId": "5dfff82c34c75b3436e7e03370e220e4693d39026fee315e6db7b7815305df4a",
-  "proof": {
-    "type": "JcsEd25519Signature2020",
-    "verificationMethod": "#sign-0",
-    "signatureValue": "3TG8LiXbWPcH3ecg4is5mswENVuQBakhv8R5TMXopHgPg578oLoczySvZMsEdjRHYgJGihK2VHEqyCHPjQyXW61m"
-  }
-}
+did:iota:iota:0xe4edef97da1257e83cbeb49159cfdd2da6ac971ac447f233f8439cf29376ebfe
+did:iota:0xe4edef97da1257e83cbeb49159cfdd2da6ac971ac447f233f8439cf29376ebfe
 ```
 
-### Anatomy of the Proof object
-
-Following the proof format in the [Verifiable Credential standard](https://www.w3.org/TR/vc-data-model/#proofs-signatures), at least one proof mechanism, and the details necessary to evaluate that, MUST be expressed for a DID Document uploaded to the Tangle. 
+### IOTA-Tag
+An IOTA-tag is a hex-encoded `Alias ID`. The `Alias ID` itself is a unique identifier of the alias, which is the BLAKE2b-256 hash of the Output ID that created it.
+This tag identifies the Alias Output where the DID Document is stored, and it will not be known before the generation of the DID since it will be assigned when the Alias Output is created.
 
-The proof object is an embedded proof that contains all information to be verifiable. It contains one or more cryptographic proofs that can be used to detect tampering and verify the authorship of a DID creation or update. It mostly follows LD-Proofs standard.
+### Anatomy of the State Metadata
 
-**Type**
+In the `State Metadata` of the Alias Output must be a byte packed payload with header fields as follows:
 
-The proof object MUST include a `type` property. This property references a verification method type's signature algorithm, as standardized in the [DID spec registries](https://www.w3.org/TR/did-spec-registries/#verification-method-types) or standardized via the method specification. 
+| Name          | Type         | Description                                                        |
+|---------------|--------------|--------------------------------------------------------------------|
+| Document Type | ByteArray[3] | Set to value **DID** to denote a DID Document.                     |
+| Version       | uint8        | Set value **1** to denote the version number of this method        |
+| Encoding      | uint8        | Set to value to **0** to denote JSON encoding without compression. |
+| Payload       | ByteArray    | Payload encoded according to `Encoding`.                           |
```

Comment:
```suggestion
| Payload       | ByteArray    | A DID document where every occurrence of the DID from the document's `id` field is replaced by `did:0:0`. It must be encoded according to `Encoding`.                           |
```

Reply:
the payload is explained under the table, it has the document and the metadata too.

Reply:
Yeah, I'm aware, but I primarily wanted everything in one place, so the important information of having replace the `did:0:0` isn't almost hidden under the table in a different place.